fhir2dataset/parser.py

```diff
@@ -0,0 +1,193 @@
+import re
+from collections import defaultdict, OrderedDict
+
+
+PREFIX = [" eq ", " ne ", " gt ", " lt ", " ge ", " le ", " sa ", " eb ", " ap "]
```

Comment:
I wonder if we need the spaces (can't we use .trim()? )

Reply:
As mentioned above, the new function manages it